src/utils/SizeAndPositionUtils.ts

```diff
@@ -0,0 +1,41 @@
+import {Component} from 'react';
+import {Dimensions} from 'react-native';
+
+class SizeAndPositionUtils extends Component {
+  static androidButtonPosition() {
+    return (
+      Dimensions.get('screen').height -
+      Math.floor(Dimensions.get('screen').height / 100) * 80
+    );
+  }
+
+  static iosButtonPosition() {
+    return (
+      Dimensions.get('screen').height -
+      Math.floor(Dimensions.get('screen').height / 100) * 85
+    );
+  }
+
+  static addToCartIOSButtonPosition() {
+    return (
+      Dimensions.get('screen').height -
+      Math.floor(Dimensions.get('screen').height / 100) * 105
+    );
+  }
+
+  static addToCartAndroidButtonPosition() {
+    return (
+      Dimensions.get('screen').height -
+      Math.floor(Dimensions.get('screen').height / 100) * 105
+    );
+  }
+
+  static ProcessToPaymentButtonPosition() {
+    return (
+      Dimensions.get('screen').height -
+      Math.floor(Dimensions.get('screen').height / 100) * 110
```

Comment:
It seems that 

```
      Dimensions.get('screen').height -
      Math.floor(Dimensions.get('screen').height / 100)
```
Part is copied too many times. Ideally it should be a solitary method, result of which is just multiplied by a specified factor.